modules/gdscript/gdscript_editor.cpp

```diff
@@ -1271,6 +1271,14 @@ static void _find_identifiers(const GDScriptParser::CompletionContext &p_context
 		}
 		r_result.insert(option.display, option);
 	}
+
+	// Global scripts
```

Comment:
I would change this comment to `// Global classes`, but it's a small nitpick.

Reply:
Yeah I agree; I'll update it now